config.yml

```diff
@@ -37,6 +37,9 @@ redis_activity_consumers: 8
 # Number of concurrent connections to Redis in the connection pool.
 redis_connection_pool_size: 20
 
+# Max number of seconds BLPOP waits before returning anyway. Must be a positive integer. 0 means wait indefinitely.
+redis_blpop_timeout: 10
```

Comment:
I think this should be 0, not sure about the polling implementation but either you repoll on timeout or just blpop 0, also make sure to use a different connection for this since the command will block the whole connection so you can't run any other commands on it!